### PR TITLE
Add promtail on boot servers

### PR DESCRIPTION
**Pull request by toshipp**
Signed-off-by: Toshikuni Fukaya <toshikuni-fukaya@cybozu.co.jp>

### CODE REVIEW

File: progs/promtail/conf.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package promtail
+
+import (
+	"io"
+)
+
+//go:embed promtail.yaml
+var promtailYaml string
```
Comment (ymmt2005) [inline progs/promtail/conf.go:8]: can be `[]byte`.
```suggestion
var promtailYaml []byte
```

File: progs/promtail/conf.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package promtail
+
+import (
+	"io"
```
Comment (ymmt2005) [inline progs/promtail/conf.go:4]: require `embed`.
```suggestion
	_ "embed"
	"io"
```

**Review by llamerada-jp — COMMENTED** [review]
LGTM

**Review by llamerada-jp — APPROVED** [review]
LGTM

**Pull request merged**